packages/discord-player/src/clientadapter/IClientAdapter.ts

```diff
@@ -1,3 +1,4 @@
+import { ChannelType } from 'discord-api-types/v9';
```

Comment:
lets use v10 🙂 
```suggestion
import { ChannelType } from 'discord-api-types/v10';
```

Reply:
Is this outdated? Seems latest uses v10

Reply:
I think you reviewed outdated code as I already updated all to v10. This is resolved. I agree we should use v10.